CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## [0.19.4] - 2022-04-12
+
+### Fixed
```

Comment:
Looks like this should be in unreleased section?

Reply:
fixed.